android/app/google-services.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
please add to `.gitignore` and send me this file manually so I can add as github secret
Also - why this file was changed?